corehq/apps/app_manager/suite_xml/utils.py

```diff
@@ -126,10 +126,9 @@ def is_valid_results_instance_name(app, instance_name):
     from corehq.apps.app_manager.suite_xml.post_process.remote_requests import (
         RESULTS_INSTANCE,
         RESULTS_INSTANCE_INLINE,
-        RESULTS_INSTANCE_BASE
     )
 
     valid_instance_names = {RESULTS_INSTANCE, RESULTS_INSTANCE_INLINE}
-    valid_instance_names.update(f'{RESULTS_INSTANCE_BASE}{module.search_config.get_instance_name()}'
+    valid_instance_names.update(f'{module.search_config.get_instance_name()}'
```

Comment:
nit: you can also drop the `f'{}'` from these now

Reply:
[4920797](https://github.com/dimagi/commcare-hq/pull/33635/commits/4920797c97538e25c9ae711ef5dc9b6cc93cd3f9)